_data/investing.yml

```diff
@@ -50,7 +50,8 @@ websites:
     url: https://www.fidelity.com
     img: fidelity.png
     tfa:
-      - software
+      - proprietary
+      - sms
       - hardware
     doc: "/notes/fidelity/"
```

Comment:
```suggestion
    doc: https://www.fidelity.com/security/how-two-factor-authentication-works
```

Reply:
@mathstuf Ben,
please replace this link to the page of fidelity and delete the `notes/fidelity.html` files.
Thanks!
// Kai